source/schema.txt

```diff
@@ -335,6 +335,28 @@ To delete a location filter from the map:
 
 .. |query-build-type| replace:: :ref:`build-query`
 
+Troubleshooting
+---------------
+
+If the analysis of your schema times out, it might be because the 
+collection you are analyzing is very large and the operation is stopped 
+before the analysis is complete. Increase the value of ``maxTimeMS`` 
+to allow the operation time to complete.
+
+To increase the value of ``maxTimeMS``:
+
+1. In the query bar, expand :guilabel:`Options`.
+
+   .. figure:: /images/compass/schema-query-options.png
+      :alt: The Options button is on the right side of the query bar, 
+            next to the green Analyze button.
+
+2. Increase the value of ``maxTimeMS`` to accommodate your collection. 
+   The value you should use depends on your collection.
```

Comment:
I feel like this is really vague, but I also suspect there's no good general guidance to give on this subject so I don't really know if it can be improved at all. Maybe give some kind of ballpark estimate, like "``maxTimeMS`` defaults to 500ms, but large collections can take multiple seconds to analyze"?

Reply:
Agreed, that was too vague - going with something similar to your suggestion. I'll call this part out in tech review to see if there's a more concrete recommendation to make.